### DISCUSSION TITLE
How to contribute to Gowebly?

### DISCUSSION THREAD

**Discussion by yaten2302** [gowebly/gowebly#93] [Q&A] [answered]
Hey @gowebly team!!
I'm a new contributor here and quite familiar with Golang and I'm also an active open source contributor(contributed to - Keploy/keploy, Aviga/Taipy). Could you please guide that how can I get started by contributing to Gowebly?

Thanks

**Comment by koddr** [comment]
Hi, @yaten2302 

If you want to contribute to the project, please read the [contributing guide](https://github.com/gowebly/gowebly/blob/main/CONTRIBUTING.md) first.

**Reply by yaten2302** [reply]
Hey @koddr , yeah! I've read the contribution guide. Actually, I wanted to ask that in the ReadMe file, it was mentioned that volunteers are required for this project. 

I would like to volunteer for this. I've experience in React, TypeScript, and JavaScript and am currently learning Golang. I can make a website for the project, and also, I'll try to contribute to the backend (written in Go).
Could you please let me know about the same, if volunteers are require, then I'm up for this task👍

**Reply by yaten2302** [reply]
Any update on this @koddr :)

**Reply by koddr** [reply, marked as answer]
@yaten2302 I don't really understand your questions.

Because we're on an open source project site, not on LinkedIn or something. You can do whatever you think is necessary around the Gowebly project, you don't need to ask "what should I do?" from the authors. 

Everything you do for the project should benefit all Gowebly CLI users. 
This is the main rule 😉
